tests/commands/canary.py

```diff
@@ -16,3 +16,9 @@ def test_cmd_canary(self):
         res = gdb_start_silent_cmd("canary", target=_target("canary"))
         self.assertNoException(res)
         self.assertIn("The canary of process", res)
+
+        if platform.machine() == "x86_64":
```

Comment:
Possibly, but I think overwriting the `AT_RANDOM` value in other cases would make the test a bit misleading, since the canary isn't really being updated, just the original value it was derived from

Reply:
Right. We should add a note herse, then: When using the auxv thing, then the canary address can't be written to in our tests like we do here, since it's not the local canary